Data/pubtator_to_xml.py

```diff
@@ -0,0 +1,152 @@
+import gzip
+
+import tqdm
+
+
+def generate_annotation(file_object, tagged_term, index):
```

Comment:
I'm pretty sure it has less bugs than the original version, but "production ready" is a big claim... I have succesfully used it as an importer and exporter tool, but I haven't done exhaustive testing.
Scaling up was an issue for me, too: When serialising a whole Medline chunk as a collection (30k abstracts plus annotations), PyBioC proved to be a serious bottleneck both in terms of memory consumption and processing time. The situation improved with the iterative serialisation method I added (one article at a time), but the corresponding method on the input size (using `iterparse()`) is still missing, I think.